### DISCUSSION TITLE
Why can't webui find the models I downloaded?

### DISCUSSION THREAD

**Discussion by zzxgraph** [open-webui/open-webui#5106] [General]
I downloaded 3 llama3.1 models through ollama, and then installed open webui through docker. Why can't webui find the models I downloaded?

**Comment by EntropyYue** [comment]
Please ensure that the UI is correctly connected to ollama

**Comment by zzxgraph** [comment]
done.

Add a new line of Environment under [Service] in the ollama.service file, a new line!

Environment="OLLAMA_HOST=0.0.0.0:11434"

and then restart the service

**Comment by himmetozcan** [comment]
sudo sed -i '/\[Service\]/a Environment="OLLAMA_HOST=0.0.0.0:11434"' /etc/systemd/system/ollama.service
